geonode/settings.py

```diff
@@ -2144,3 +2146,14 @@ def get_geonode_catalogue_service():
 UI_REQUIRED_FIELDS = ast.literal_eval(os.getenv('UI_REQUIRED_FIELDS ', '[]'))
 
 UPLOAD_SESSION_EXPIRY_HOURS = os.getenv('UPLOAD_SESSION_EXPIRY_HOURS ', 24)
+
+# If a command name is listed here, the command will be available to admins over http
+# This list is used by the management_commands_http app
+MANAGEMENT_COMMANDS_EXPOSED_OVER_HTTP = [
+    "ping_mngmt_commands_http",
+    "updatelayers",
+    "sync_geonode_layers",
+    "sync_geonode_maps",
+    "importlayers",
+    "set_all_layers_metadata",
+]
```

Comment:
I'm thinking out loud, is it worth having this as an environment variable? it will be something like (by removing duplicates):

`MANAGEMENT_COMMANDS_EXPOSED_OVER_HTTP = defaults + os.get_env('LIST_OF_COMMANDS_FROM_ENV', [])`